master/docs/examples/git_gerrit.cfg

```diff
@@ -163,7 +166,9 @@ def gerritSummaryCB(buildInfoList, results, status, arg):
             msg += " - " + link
         else:
             msg += "."
-        msgs.append(msg)
+
+        if buildInfo['build']['properties']['shouldReport']:
```

Comment:
shouldReport is a property that you are introducing (that will not exist on others infras)

I would prefer that you dont change the default